_includes/home/_content-menu.html

```diff
@@ -0,0 +1,15 @@
+<div class="container">
+    <div class="col-md">
+        <section class="content-menu">
+            <p class="menu-title soft-left font-family-serif text-uppercase">Explore All of Our Content</p>
+                <ul style="list-style: none; padding-left: 0px; padding-top: 4px;"> 
```

Comment:
I don't think these inline styles are necessary. you could use these classes `list-unstyled hard-left soft-quarter-top`